README.md

```diff
@@ -83,6 +83,7 @@ END
   - `name` - a string with instance name
   - `uri`  - a string with uri to bind to, for example: `0.0.0.0:11211` (only TCP is supported now)
   - `opts` - a table with options, the list of possible options is described in the configuration section
+* `local slab = memcached.slab()` - show information about slab arena and quota.
```

Comment:
Nit: I would place it before `memcached.create()`, which starts a description of things related to an instance.